test/functions.php

```diff
@@ -65,4 +61,40 @@ public function testErrorMakePlantEntryTileEmptyInput() {
         $actual = makePlantEntryTile([]);
```

Comment:
```suggestion
        $actual = makePlantEntryTile();
```

Reply:
again if this is testing an array ignore me, just trying to find things because it all looks good.